docs/banzai/index.rst

```diff
@@ -2,80 +2,136 @@
+We currently use the header the keyword ``BIASSEC`` to identify the overscan region. If this keyword is set to
 "Unknown", then we simply skip subtracting the overscan. We estimate a single overscan value for the whole image
 (rather than row by row).
-This is saved in the header under the keyword 'BIASLVL'. If there is no overscan section, this value is derived
+This is saved in the header under the keyword ``BIASLVL``. If there is no overscan section, this value is derived
 from the average value from the bias frames.
 
-The Sinistro frames do have overscan regions, but the overscan is currently removed by the preprocessor.
-
 
 Crosstalk
 =========
-Currently, only Sinstro images are read out using multiple amplifiers. The Sinsitro frames do have significant
+Currently, only Sinistro images are read out using multiple amplifiers. The Sinistro frames do have significant
 crosstalk between amplifiers, but this is currently removed by the preprocessor. The crosstalk is removed using
 linear coefficients that relate each quadrant to every other quadrant.
 
 
+Gain
+====
+All pixels in the frame are multiplied by the gain, using the ``GAIN`` header keyword.
+
+
 Mosaic
 ======
-Again, only the Sinstro frames currently read out with multiple amplifiers so mosaicing the amplifiers
-is only required for Sinsitros. This is currently done by the preprocessor.
+Again, only the Sinistro frames currently read out with multiple amplifiers so mosaicing the amplifiers
+is only required for Sinsitros.
 
 
 Trim
 ====
-After the overscan is subracted, the bias, dark, flat-field, and science images are trimmed
-based on the TRIMSEC header keyword.
+After the overscan is subtracted, the bias, dark, flat-field, and science images are trimmed
+based on the ``TRIMSEC`` header keyword.
+
+
+Bias Subtraction
+================
+Full frame bias images are subtracted from each of the darks, flat field images, and science frames.
+The master bias frame that was taken closest in time to the current data will be used.
+This will add a few counts of noise to each pixel, but it solves two problems. First, if there is systematic
+structure in the read out, this will be removed. Second, this will remove the bias values for images
+that do not have an overscan region.
+
+If no bias frame exists for this instrument, the data will not be reduced and an exception will be
+raised.
+
+
+Dark Subtraction
+================
+Full-frame master dark frames, scaled to the exposure time of the frame,
+are subtracted from all flat-field and science images. The most recent
+master dark frame is used. Often this is taken on the same day. If no dark frame exists for this
+instrument, the data will not be reduced and an exception will be raised.
+
+
+Flat Field Correction
+=====================
+Master flat field images (normalized to unity) are divided out of every science frame. The most recent
```

Comment:
Should we mention that only the middle quarter of the image is used for the normalization?

Reply:
Good point, added.